src/components/slider/slider.scss

```diff
@@ -49,16 +74,22 @@ md-slider {
 
   height: $slider-height;
   position: relative;
-  display: block;
   margin-left: 4px;
   margin-right: 4px;
   padding: 0;
+  justify-content: center;
+  display: flex;
```

Comment:
@EladBezalel - 

```html
<md-slider ....>
   <div class="md-slider-wrapper">
 
    ....
   </div>
</md-slider>
```

As @topherfangio  mentioned, you should make the mdSlider component `display: inline-block` so it can be easily used/integrated with other elements. And then use the Layout API for the `md-slider-wrapper` and its children.

![screen shot 2016-02-06 at 7 08 02 am](https://cloud.githubusercontent.com/assets/210413/12866785/ba4a331e-cca0-11e5-9ad1-90b30640ee45.png)